chain/epoch-manager/src/adapter.rs

```diff
@@ -956,6 +963,31 @@ impl EpochManagerAdapter for EpochManagerHandle {
         }
     }
 
+    fn verify_chunk_endorsement(
+        &self,
+        chunk_header: &ShardChunkHeader,
+        endorsement: &ChunkEndorsement,
+    ) -> Result<bool, Error> {
+        let epoch_id = self.get_epoch_id_from_prev_block(chunk_header.prev_block_hash())?;
+        let chunk_validators = self.get_chunk_validators(
+            &epoch_id,
+            chunk_header.shard_id(),
+            chunk_header.height_created(),
+        )?;
+        if !chunk_validators.contains_key(&endorsement.account_id) {
+            return Err(Error::NotAValidator);
+        }
+        let (validator, is_slashed) = self.get_validator_by_account_id(
```

Comment:
We don't have slashing implemented and I don't know if slashing makes any sense for chunk endorsement validation (e.g. if a validator is hypothetically slashed, we don't make another chunk validator shard assignment calculation so we would still be expecting this validator to validate the chunk but then we ignore its endorsement...) I wonder if we should just ignore is_slashed. It's dead code anyway.